libs/blocks/merch/merch.css

```diff
@@ -15,7 +15,7 @@ span.placeholder-resolved[data-template="strikethrough"], span.price.price-strik
   visibility: inherit;
 }
 
-a[is='checkout-link'] > span {
+a[is='checkout-link']:not(.text-link) > span {
```

Comment:
I think you can use `.con-button` here, since it is already added to CTA like co links.

Reply:
Right! Fixed.